### PR TITLE
[new] Add infinite-ajax-scroll w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by ignatiusmb**
# Pull request template for new library

Pull request for issue: #13655 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/webcreate/infinite-ajax-scroll
* Official website (optional, not the repository): https://infiniteajaxscroll.com/
* NPM package url (optional): https://www.npmjs.com/package/@webcreate/infinite-ajax-scroll
* License and its reference: AGPL-3.0-only
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 57
  * Count of stars: 725
  * Count of forks: 341
* NPM download stats (optional):
  * Downloads in the last day: 
  * Downloads in the last week: 180
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by fieg** [comment]
@MattIPv4 any ideas why the test is failing?

**Comment by MattIPv4** [comment]
The CI doesn't work anymore, it can be ignored.

File: ajax/libs/infinite-ajax-scroll/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "infinite-ajax-scroll",
+  "title": "Infinite Ajax Scroll",
+  "description": "Turn your existing pagination into infinite scrolling pages with ease",
+  "license": "AGPL-3.0-only",
+  "homepage": "https://infiniteajaxscroll.com",
+  "repository": "github:webcreate/infinite-ajax-scroll",
```
Comment (MattIPv4) [inline ajax/libs/infinite-ajax-scroll/package.json]:
```suggestion
  "repository": {
    "type": "git",
    "url": "git+https://github.com/webcreate/infinite-ajax-scroll.git"
  },
```

File: ajax/libs/infinite-ajax-scroll/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "infinite-ajax-scroll",
+  "title": "Infinite Ajax Scroll",
```
Comment (MattIPv4) [inline ajax/libs/infinite-ajax-scroll/package.json]:
```suggestion
```
Attr not needed

**Comment by ignatiusmb** [comment]
@MattIPv4 change request resolved, thanks!

**Comment by fieg** [comment]
Hi @MattIPv4, it seems like Infinite Ajax Scroll still isn't showing on cdnjs.com. Any thoughts?

**Comment by MattIPv4** [comment]
I believe the auto-update bot currently isn't picking up libraries added via a single package.json file. I'm hoping this will be the next thing that @xtuc will be able to work on :)

**Comment by fieg** [comment]
Is there an alternative approach to "added via a single package.json file" we could take?

**Comment by MattIPv4** [comment]
You can create a PR to manually add files for all versions that are missing, but that is normally discouraged as it requires a lot of work on our end to verify everything before we can merge it. 

We are currently working to get the auto-update bot rewritten and ready to work with single package.json additions.

**Pull request merged**